financialaid/constants.py

```diff
@@ -0,0 +1,224 @@
+"""
+Constants for financialaid
+"""
+
+
+DEFAULT_INCOME_THRESHOLD = 100000
+COUNTRY_INCOME_THRESHOLDS = {
```

Comment:
This was the issue I mentioned on hipchat a few days ago.  The files sent to us by the econ department used the word bank codes, not the ISO 3166-3 codes. which are mostly overlapping, except for the set you are seeing here.  We corrected these manually when we converted to ISO-31662 codes to be consistent with the rest of the system